operators/src/upgrade/bin/main.rs

```diff
@@ -26,11 +34,23 @@ async fn main() -> Result<(), Error> {
             .service(service::get_upgrade)
     };
 
-    HttpServer::new(app)
-        .bind(("127.0.0.1", 8080))
-        .expect("Unable to bind address")
-        .run()
-        .await
-        .expect("Unable to run the server");
-    Ok(())
+    let app_server = HttpServer::new(app)
+        .bind(("0.0.0.0", UPGRADE_OPERATOR_INTERNAL_PORT))
+        .map_err(|error| {
+            error!(
+                ?error,
+                "failed to bind API to socket address 0.0.0.0:{}", UPGRADE_OPERATOR_INTERNAL_PORT
+            );
+            Error::from(error)
+        })?;
+
+    // Start Upgrade API.
+    info!(
+        "Starting to listen on port {}",
+        UPGRADE_OPERATOR_INTERNAL_PORT
+    );
+    app_server.run().await.map_err(|error| {
```

Comment:
Should probably configure the number of worker, PTAL at the control-plane's rest server.

Reply:
Yes... 1 worker should be enough.